packages/sanity/src/core/environment/hasSanityPackageInImportMap.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * The presence of the sanity module in the importmap script
+ * indicates that the studio was probably built with an auto-updates flag.
+ */
+
+/** @internal */
```

Comment:
Wondering if these can be unified to make the description attached to the function, that way we get a little more context when using it in IDEs etc?

```suggestion
/**
 * Returns wether or not there is a `sanity` entry in an import map in the current document,
 * which usually means that this studio is "auto updating".
 * @internal
 */
```